src/desktop/ElectronUpdater.ts

```diff
@@ -215,6 +214,17 @@ export class ElectronUpdater {
 		}
 	}
 
+	private async checkUpdateThrottleTime() {
+		// After 5 unsuccessful attempts to update, we start throttling the user
+		// to avoid too many calls to our servers
+		if (this.errorCount >= 5) {
+			// Half of a second * errorCount
+			const throttleTime = 500 * this.errorCount
+			this.logger.debug(`Throttling user ${throttleTime} - Try N ${this.errorCount}`)
```

Comment:
I would just use `debounceStart` with some fixed time but I think yours should also be good.

I would change the message to `Auto Update: throttling manual update attempt # ${this.errorCount} by ${throttleTime}`